src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/ProjectServices/VsManagedLanguagesProjectSystemServices.cs

```diff
@@ -196,6 +197,16 @@ private static string GetReferenceMetadataValue(ProjectReference reference, stri
 
         private static LibraryDependency ToPackageLibraryDependency(PackageReference reference, bool isCpvmEnabled)
         {
+            if (string.IsNullOrWhiteSpace(reference.Version))
```

Comment:
This fails the operation, while on the commandline it succeeds with a warning. 

Not sure this is the right solution. 

You probably just need to add a null version and the rest of the reporting will be handled like it is on the commandline.

Reply:
@nkolev92 
Can you tell me where is code for command line of this one?
Is it `NuGet.Build.Tasks.Console` ? If yes can you give me sample `Application arguments` for this project?

Reply:
https://github.com/NuGet/NuGet.Client/blob/a63a08cb7dcf6a873f353cbcc27f78335e20d187/src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/MSBuildRestoreUtility.cs#L623

You should try tracing back to find the equivalent. 

For example, try to trace back where the LibraryDependency is created. Rely on https://github.com/NuGet/NuGet.Client/blob/dev/docs/project-overview.md for understanding what each project contains

Reply:
@nkolev92 
Can you tell me what is this 4-th parameter is expecting for `NuGet.Build.Tasks.Console` ? I need it to start the this project?

![image](https://user-images.githubusercontent.com/8766776/102168192-add44500-3e44-11eb-8112-f9a9c13789c4.png)

Reply:
@nkolev92 
I mean can you share your? 
![image](https://user-images.githubusercontent.com/8766776/102230611-f66e1b80-3ea1-11eb-961a-b56100e7b98a.png)

Reply:
https://github.com/NuGet/NuGet.Client/blob/dev/docs/debugging.md#restore-in-msbuild

That's the static graph wrapper, which I'm pretty confident you shouldn't need. Let me know if you think otherwise. 

Note that the link that pointed to MSBuildRestoreUtility covers the standard msbuild restore part.

Reply:
Ok, removed exception throw and  changed to VersionRange.All.